lookout/style/format/analyzer.py

```diff
@@ -522,16 +531,11 @@ def _group_line_nodes(
         for line_no, (line_y, line_y_pred, line_vnodes_y, line_rule_winners) in result:
             line_vnodes = []
             for vnode in new_vnodes:
-                if vnode.start.line > line_no:
-                    if (line_vnodes and line_vnodes[-1].y is not None and
-                            CLASS_INDEX[CLS_NEWLINE] in line_vnodes[-1].y):
```

Comment:
this complex condition was needed to join lines if we remove newline but as I said we cannot provide a good suggestion in such case.